### DISCUSSION TITLE
Seems like all the FP software has gone AWOL on Sourceforge

### DISCUSSION THREAD

**Discussion by Lotus49b** [freeplane/freeplane#954] [Help]
Been trying to pick up a newer Preview version, but nothing is showing. It was all there earlier today.

Is the site closed for maintenance?

**Comment by dpolivaev** [comment]
I have no idea, so I submitted a ticket to source forge support

https://sourceforge.net/p/forge/site-support/24239/

**Reply by dpolivaev** [reply]
https://sourceforge.net/p/forge/site-support/ contains a lot of similar tickets, so I believe it is a Sourceforge general failure which should be fixed soon

**Reply by Lotus49b** [reply]
Thanks!
